src/Mapbender/CoreBundle/Asset/ApplicationAssetService.php

```diff
@@ -223,6 +223,7 @@ protected function getFrontendBaseAssets(string $type): array
                     '@MapbenderCoreBundle/Resources/public/init/element-sidepane.js',
                     '@MapbenderCoreBundle/Resources/public/widgets/toolbar-menu.js',
                     '/components/datatables/media/js/jquery.dataTables.min.js',
+                    'components/datatables/media/js/dataTables.bootstrap4.min.js',
```

Comment:
does the bootstrap4 template work reliably for us? (we have bootstrap 5 now)

The composer package of datatables is once again quite outdated, the newest available version is 1.10.21 from 10 years ago, the current repo is https://github.com/DataTables/DataTablesSrc/tags

Also, you should probably add a / at the start or remove it from the line above, but I'd have it consistent

Reply:
@Phocacius I fixed the assets path. It works reliable, even with datatables' Bootstrap 4 version. You may merge now. Considering the datatables update, I suggest we stick with the old version for now and run the update later on in a second step.